push/views.py

```diff
@@ -0,0 +1,35 @@
+from django.views.generic import TemplateView
+
+from push.models import PushApplication
+from push.forms import VapidValidationForm
+
+class PushApplicationDetails(TemplateView):
+    template_name = 'push/details.html'
+
+    def get_context_data(self, **kwargs):
+        push_app = PushApplication.objects.get(pk=self.kwargs['pk'])
+
+        context = super(PushApplicationDetails, self).get_context_data(**kwargs)
+        context.update({
+            'id': push_app.id,
+            'name': push_app.name,
+            'vapid_key_status': push_app.vapid_key_status,
+            'vapid_key_token': push_app.vapid_key_token,
+        })
+
+        return context
+
+class ValidatePushApplication(TemplateView):
```

Comment:
To complete the validate workflow, we'll need to add a view that receives the `POST`, calls the model method to verify the signed token and update the database, and then returns the user to this `GET` version of the view. Can file as a follow-up issue too.